api/users/models.py

```diff
@@ -469,6 +469,41 @@ class HubspotTracker(models.Model):
         related_name="hubspot_tracker",
         on_delete=models.CASCADE,
     )
-    hubspot_cookie = models.CharField(unique=True, max_length=100, null=False)
+    hubspot_cookie = models.CharField(
+        unique=True,
+        max_length=100,
+        null=True,
+        blank=True,
+    )
+    utm_source = models.CharField(max_length=256, null=True, blank=True)
+    utm_medium = models.CharField(max_length=256, null=True, blank=True)
+    utm_campaign = models.CharField(max_length=256, null=True, blank=True)
+    utm_term = models.CharField(max_length=256, null=True, blank=True)
+    utm_content = models.CharField(max_length=256, null=True, blank=True)
```

Comment:
This is something that we're only ever going to be worried about in SaaS, and hence we can build assuming that we only need to support postgres. Based on this, should we instead be using a JSONField here for `utm_data` instead of creating a field for each `utm` property. 

Sorry, I should probably have raised this on my first review... 

Keen to get @khvn26's opinion here too based on his recent experience using HStore field.